session-jdbc-security/README.md

```diff
@@ -0,0 +1 @@
+Tests the session persistence and cookie customization from Spring Session together with Spring Session JDBC and Spring Security.
```

Comment:
I don't quite get why this smoke test has Security in it. Is there something in session / security which has to be tested in combination? If not, I would be in favor of removing security from that smoke test.